readme.md

```diff
@@ -44,11 +45,11 @@ React can be thought of as the "Views" layer.
 <details>
   <summary><strong>What is the role of a "view" in a front-end Javascript application?</strong></summary>
 
-  > The visual template the user sees, populated with data from our models.
+  > The visual template the user sees, often populated with data from our models.
 
 </details>
 
-React can be used agnostically throughout your stack. It's role is just to use data to render a UI. This means that React can also coexist with other Javascript frameworks. Let them handle the models and controllers, and have React sort out the views.
+React can be used agnostically throughout your stack. It's role is just to use data to render a UI. This means that React can also co-exist with other Javascript frameworks. Let the other frameworks handle the models and controllers, and have React sort out the views.
```

Comment:
Are there any resources / examples we can provide students of React being used with other frameworks? Don't need to talk about it in class but some quick links here could be nice.